torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8190,6 +8190,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestProxyTensorOpInfo', 'test_make_fx_symbolic_exhaustive'),
                # aten.uniform was not decomposed
                DecorateInfo(unittest.expectedFailure, 'TestDecomp', 'test_quick'),
+               # Can not test non deterministic ops between devices
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_compare_cpu'),
```

Comment:
The expected failures due to non-deterministic behavior should probably all be `skip` as coincidental successes would cause the test to be flaky. You have done this in some places but many still use xfail.